src/components/Pages/Search/index.js

```diff
@@ -1,46 +1,160 @@
+    // setSearchResults(filteredSearch)
+  }
 
   return (
     <div>
+
       <Head>
-        <title> {title} </title>
+        <title> {intl.formatMessage(i18n.search)} </title>
       </Head>
 
-      <PageHeader> {title} </PageHeader>
+      <PageHeader contentType={'search'}>
+        {intl.formatMessage(i18n.search)}
+        <div className="coa-search_bar_container">
+          <input
+            id="coa-search_input"
+            className="coa-search_inputs"
+            onChange={()=>searchKeyInput(event)}
+            value={searchString}
+          />
+
+          <button
+            className="coa-search_button"
+            onClick={()=>searchButtonPressed('click')}
+          >
+            {intl.formatMessage(i18n.search)}
+          </button>
+        </div>
+      </PageHeader>
+
+      <div id="coa-search_loading_wheel" className="coa-loading_wheel"></div>
+
+      <div className="wrapper container-fluid">
+        <div className="row">
+
+          <div id="coa-search_results" className="col-xs-12 col-md-8">
 
-      <br /><h1> &nbsp; &nbsp; 🚧... Under Construction 🏗</h1>
+            {searchedTerm && searchResults.length < 1 && (
+              <NoResults />
+            )}
 
-      {/* - - - TO DO: Finish Janis Search Page - - - - - */}
+            <div className="coa-search_results-total">
+              {searchedTerm && searchResults.length > 0 && (
+                <span>
+                  {searchResults && searchResults.length}
+                  &nbsp;{intl.formatMessage(i18n.results)} <em>"{searchedTerm}"</em>
+                </span>
+              )}
+            </div>
 
-      { searchIndex && searchIndex.map( (page, i) => (
-        <SearchResult
-          page={page}
-          key={i}
-        ></SearchResult>
-      )) }
+            { searchResults && searchResults.map( (page, index) => (
+              <SearchResult
+                page={page}
+                key={index}
+              />
+            )) }
 
-      {/* - - - TO DO: Finish Janis Search Page - - - - - */}
+          </div>
+
+        </div>
+      </div>
+
+
+      {/*
+        TODO: PAGINATION ( https://github.com/cityofaustin/techstack/issues/4358 )
+        NOTE: It would be too much to handle all in one with this component like we've
+        done on other pages.
+        - Make paginationContainer.js
+          <PaginationContainer
+            results={searchResults}
+          />
+      */}
 
     </div>
   )
 }
 
+const NoResults = function() {
+
+  return (
+    <div>
+      <div className="coa-search_results-total">
+        0 results
+      </div>
+      <h2 className="coa-search_results-zero-message">
+        There are no matching results. Improve your search results by:
+      </h2>
+      <div className="coa-search_results-zero">
+        • double-checking your spelling <br />
+        • using fewer keywords <br />
+        • searching for something less specific <br />
+      </div>
+    </div>
```

Comment:
just a note to i18nify this

Reply:
Actually made an issue for it as well
- https://github.com/cityofaustin/techstack/issues/4423